<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Computer Status Neo</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabSingleComputer">
     <attribute name="title">
      <string>Single Computer Mode</string>
     </attribute>
     <widget class="QPushButton" name="buttonExecuteAction">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>420</y>
        <width>169</width>
        <height>64</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>64</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&amp;Execute Action</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="dropdownActions">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>440</y>
        <width>271</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>System Info</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Reactivate Windows License</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Get AD Join Status</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Reinstall Office 365</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Install Printer...</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Shut Down</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Restart</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Run SFC and DISM</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="labelActions">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>440</y>
        <width>47</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Actions:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>751</width>
        <height>401</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,7">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <layout class="QVBoxLayout" name="leftPane" stretch="0,0,0,0,0,0,0,0,0">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="labelComputer">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>26</height>
            </size>
           </property>
           <property name="text">
            <string>Computer to Query</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="inputComputer">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonNetDB">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>&amp;NetDB</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonRemoteDesktop">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Remote &amp;Desktop</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonRemoteAssistance">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Remote &amp;Assistance</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonCDollarAdminShare">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>&amp;c$ Admin Share</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonComputerManagement">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Computer &amp;Management</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonReverseShell">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Reverse &amp;Shell</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="rightPane">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <item>
          <widget class="QLabel" name="labelResult">
           <property name="text">
            <string>Result</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="textResult">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <zorder>buttonExecuteAction</zorder>
     <zorder>horizontalLayoutWidget</zorder>
     <zorder>labelActions</zorder>
     <zorder>dropdownActions</zorder>
     <zorder>labelRunningAs</zorder>
     <zorder>buttonSwitchUser</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>771</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="8,1">
     <property name="rightMargin">
      <number>4</number>
     </property>
     <item>
      <widget class="QLabel" name="labelRunningAs">
       <property name="text">
        <string>Running as: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSwitchUser">
       <property name="maximumSize">
        <size>
         <width>83</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Switch &amp;User...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
